theories/_CoqProject

```diff
@@ -9,37 +9,38 @@
 # ./Algorithmic/Typing/Definitions.v
 # ./Algorithmic/Typing/Lemmas.v
 ./Core/Base.v
-# ./Core/Completeness.v
-# ./Core/Completeness/Consequences/Rules.v
-# ./Core/Completeness/Consequences/Types.v
-# ./Core/Completeness/ContextCases.v
-# ./Core/Completeness/FunctionCases.v
-# ./Core/Completeness/FundamentalTheorem.v
-# ./Core/Completeness/LogicalRelation.v
-# ./Core/Completeness/LogicalRelation/Definitions.v
-# ./Core/Completeness/LogicalRelation/Lemmas.v
-# ./Core/Completeness/LogicalRelation/Tactics.v
-# ./Core/Completeness/NatCases.v
-# ./Core/Completeness/SubstitutionCases.v
-# ./Core/Completeness/SubtypingCases.v
-# ./Core/Completeness/TermStructureCases.v
-# ./Core/Completeness/UniverseCases.v
-# ./Core/Completeness/VariableCases.v
+./Core/Completeness.v
+./Core/Completeness/Consequences/Rules.v
+./Core/Completeness/Consequences/Types.v
+./Core/Completeness/ContextCases.v
+./Core/Completeness/EqualityCases.v
```

Comment:
if I am seeing it correctly, you didn't include this file at all.

Reply:
They are naturally available again as we don't need to change any for them.

Reply:
You don't have EqualityCases. This must be a new file.

Reply:
Ah, I didn't commit the file! what...